_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -499,17 +641,81 @@ Theoretically you can send any kind of data in a Relationship Template. However,
-| -------- | ----------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| response | [`Response`](#response) | The Response to the Request that was contained in the [RelationshipTemplateContent](#relationshiptemplatecontent) (either in the `onExistingRelationship` property or in the `onNewRelationship` property). |
+| Name     | Type                                         | Description                                                                                                                                                                                                 |
+| -------- | -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| @type    | `"RelationshipCreationChangeRequestContent"` |                                                                                                                                                                                                             |
+| response | [`Response`](#response)                      | The Response to the Request that was contained in the [RelationshipTemplateContent](#relationshiptemplatecontent) (either in the `onExistingRelationship` property or in the `onNewRelationship` property). |
 
 ## Mail
 
 A Mail can be sent as the content of a [Message](#message). It is comparable with the classic email, so its properties should not contain any surprise.
 
 | Name    | Type                      | Description                                                                                                 |
 | ------- | ------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| @type   | `"Mail"`                  |                                                                                                             |
 | to      | `string[]`                | The enmeshed Addresses of the main recipients of this Mail.                                                 |
 | cc      | `string[]` \| `undefined` | The enmeshed Addresses that should receive a copy of this Mail, additionally to the ones specified in `to`. |
 | subject | `string`                  | The subject of the Mail.                                                                                    |
 | body    | `string`                  | The body of the Mail.                                                                                       |
+
+# Consumption
+
+## DecideRequestItemParameters
+
+In order to answer a Request, the Response must be formulated using the appropiate parameters.
```

Comment:
I think an important information is missing here. In general "answering a Request" is done via a Response. These params are only made to be an interface that is way better to consume.